app/jobs/importo/import_job.rb

```diff
@@ -20,7 +20,7 @@ def self.execute_row(attributes, index, import_id, last_attempt, bid)
         batch = Importo::SidekiqBatchAdapter.find(bid)
 
         if !import.completed? && import.can_complete? && batch.finished?
-          ImportJobCallback.new.on_complete(batch.status, {import_id: import.id})
+          ImportJobCallback.new.on_complete(:complete, {import_id: import.id})
```

Comment:
From this I guess we need to pass success here instead of complete

` ImportJobCallback.new.on_complete("success", {import_id: import.id})`